lib/teiserver/coordinator/consul_server.ex

```diff
@@ -1271,8 +1240,22 @@ defmodule Teiserver.Coordinator.ConsulServer do
 
   @spec list_players(map()) :: [T.client()]
   def list_players(%{lobby_id: lobby_id}) do
-    member_list = Battle.get_lobby_member_list(lobby_id)
+    list_members((%{lobby_id: lobby_id}))
+        |> Enum.filter(fn client -> client.player == true end)
```

Comment:
you don't need to check for `== true`, just returning `client.player` is enough.